### ISSUE TITLE
Is convAI2 test dataset unavailable?

### ISSUE THREAD

**Issue by ddehun** [facebookresearch/ParlAI#4635]
Hi. I hope to train and evaluate my response selection model on ConvAI2 dataset, but it seems that only train and validation splits are available in ParlAI platform. Is the hidden test split not publically opened, as mentioned in #1107 ?

In such a case, is there any recommended way to evaluate my (an other's) response selection model and report its performance in the paper?
I plan to use the current validation set as a test set, and make another validation set with randomly sampled responses from train set, but I'm not sure whether this is a recommended approach.
thank you for your support in advance!!

**Comment by klshuster** [comment]
Your approach sounds like an appropriate way to go about it. Nearly every paper utilizing the ConvAI2 dataset reports performance on the validation set so you should be able to compare to previous methods.

**Comment by stephenroller** [comment]
@jxmsML didnt you release this at some point?
